cmd/start/cmd.go

```diff
@@ -130,6 +130,28 @@ func enqueueMissingBlocks(exportQueue types.HeightQueue, ctx *parser.Context) {
 		panic(fmt.Errorf("failed to get last block from RPCConfig client: %s", err))
 	}
 
+	lastBlockHeightInDB, err := ctx.Database.GetLastBlockHeight()
+	if err != nil {
+		ctx.Logger.Error("failed to get last block height from database", "error", err)
+	}
+
+	// Get the start height, default to the config's height
+	startHeight := cfg.StartHeight
+
+	// Set startHeight to the latest height in database
+	// if is not set inside config.yaml file,
+	if startHeight == 0 {
+		if lastBlockHeightInDB > 0 {
+			startHeight = lastBlockHeightInDB
+		} else {
+			startHeight = 1
+		}
+	}
```

Comment:
I think this can be simplified as 
```suggestion
	if startHeight == 0 {
		startHeight = math.MaxInt64(1, lastBlockHeightInDB)
	}
```

Note: we can rename `lastBlockHeightInDB` to `lastDbBlockHeight` probably